Scripts/Python/dsntKILightMachine.py

```diff
@@ -57,6 +58,8 @@
 respDispensor = ptAttribResponder(2,"resp: KI dispensor")
 respGotKI = ptAttribResponder(3,"resp: got KI")
 sdlKILightFunc = ptAttribString(4,"sdl: KI light func")
+intKILightTime = ptAttribInt(5,"int: Length that Ki light lasts in seconds",0)
+boolKIRandom = ptAttribBoolean(6,"bool: Randomize time light last, longest time is intKILightTime, Shortest is 5s",False)
```

Comment:
I wonder if it would make more sense/be more flexible to change the `kLightTimeShort` and `kLightTimeLong` constants to attributes with the defaults that the constants currently have. IMO, the random in range operation should be a new KI light func mode (`byteKILightFunc`).